docs/content/en/docs-v1.0.x/quickstart/_index.md

```diff
@@ -10,7 +10,7 @@ PipeCD consists of two core components: The Control Plane and Piped (see [PipeCD
 
 - **The Control Plane** can be thought of as a web service application that can be installed anywhere. It provides the web UI, API endpoints, and metadata storage.
 
-- **Piped** is a lightweight agent that connects your infrastructure to the Control Plane. In PipeCD v1, Piped drops built-in platform support and uses a plugin-based model. Each plugin (an external component) implements the deployment and synchronization logic for a specific application kind, such as Kubernetes or Terraform.
+- **Piped** is a lightweight agent that connects your infrastructure to the Control Plane. In PipeCD v1, each plugin (an external component) implements the deployment and synchronization logic for a specific application kind, such as Kubernetes or Terraform.
```

Comment:
We don't need to specify plugins here; no need to show too much information at once. Keep the quickstart as quickstart is better. Also, should not mention PipeCD v1 as discussed in this comment https://github.com/pipe-cd/pipecd/pull/6321#discussion_r2488587570